frontend/public/locales/en/translation.json

```diff
@@ -1227,6 +1227,8 @@
     "Enter your vSphere cluster name": "Enter your vSphere cluster name",
     "Enter your vSphere datacenter": "Enter your vSphere datacenter",
     "Enter your vSphere default defaultDatastore": "Enter your vSphere default defaultDatastore",
+    "Enter your vSphere folder absolute path": "Enter your vSphere folder absolute path",
+    "Enter your vSphere resource pool absolute path": "Enter your vSphere resource pool absolute path",
```

Comment:
Similarly: "Enter the absolute path to your vSphere resource pool": "Enter the absolute path to your vSphere resource pool"